creating-virtual-machines/virtualized-hardware-configuration.adoc

```diff
@@ -253,6 +301,11 @@ Behaviour according to Policies:
 
 Full description about features and policies can be found https://libvirt.org/formatdomain.html#elementsCPU[here].
 
+When CPUNodeDiscovery feature-gate is enabled (as mentioned in link:#Enabling cpu compatibility enforcement[*Enabling cpu compatibility enforcement*]), 
+Kubevirt creates node selector from cpu features with format:
```

Comment:
Im not sure this whole section is necessary. we talk about the behaviour above (and mention the feature gate)

Reply:
But we don't talk there about label format.